irohad/model/converters/impl/json_command_factory.cpp

```diff
@@ -38,9 +43,9 @@ namespace iroha {
         template <typename T>
         auto operator()(T &&x) {
           auto des = makeFieldDeserializer(x);
-          return nonstd::make_optional<Amount>()
-              | des.Uint64(&Amount::int_part, "int_part")
-              | des.Uint64(&Amount::frac_part, "frac_part");
+          return nonstd::make_optional<Amount>() |
```

Comment:
plz don't do this
it will be hard to merge otherwise

Reply:
clang-format  😢